supermarq/feature_plotting.py

```diff
@@ -0,0 +1,135 @@
+from typing import Any, List, Optional, Tuple, Union
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.patches import Circle
+from matplotlib.projections import register_projection
+from matplotlib.projections.polar import PolarAxes
+
+
+def plot_benchmark(
+    data: List[Union[str, List[str], List[List[float]]]],
+    show: bool = True,
+    savefn: Optional[str] = None,
+    spoke_labels: Optional[List[str]] = None,
+    legend_loc: Tuple[float, float] = (0.75, 0.85),
+) -> None:
+    """
+    Create a radar plot of the given benchmarks.
```

Comment:
Change docstring to be in the style of Google's recommendation: https://google.github.io/styleguide/pyguide.html